### PR TITLE
Fix passing replace flag to check_stock_quantity_bulk

### PR DESCRIPTION
**Pull request by CossackDex**
I want to merge this change because it fixes bug with wrong behaviour of CheckoutLinesUpdate mutation.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4435 CheckoutLinesUpdate - it is not possible to decrease the quantity.](https://app.clickup.com/t/2549495/SALEOR-4435)

File: saleor/graphql/checkout/tests/test_checkout_lines.py [hunk 1]
```diff
@@ -491,6 +491,27 @@ def test_checkout_lines_update_channel_without_shipping_zones(
...
+
+    variables = {
+        "token": checkout.token,
+        "lines": [{"variantId": variant_id, "quantity": current_stock.quantity - 1}],
+        "channelSlug": checkout.channel.slug,
+    }
+    response = user_api_client.post_graphql(MUTATION_CHECKOUT_LINES_UPDATE, variables)
+    content = get_graphql_content(response)
+    data = content["data"]["checkoutLinesUpdate"]
+    assert not data["errors"]
```
Comment (IKarbowiak) [inline saleor/graphql/checkout/tests/test_checkout_lines.py:512]: If you update the line quantity it should be checked here.

**Review by tomaszszymanski129 — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

File: saleor/warehouse/availability.py [hunk 2]
```diff
@@ -47,6 +47,7 @@ def check_stock_quantity_bulk(
     quantities: Iterable[int],
     channel_slug: str,
     existing_lines: Iterable = None,
+    replace=None,
```
Comment (maarcingebala) [inline saleor/warehouse/availability.py:50]: `replace` is a boolean flag, so the default value should be a boolean as well.

File: saleor/graphql/checkout/tests/test_checkout_lines.py [hunk 3]
```diff
@@ -491,6 +491,27 @@ def test_checkout_lines_update_channel_without_shipping_zones(
...
+    checkout = checkout_with_item
+    line = checkout.lines.first()
+    variant_id = graphene.Node.to_global_id("ProductVariant", line.variant.pk)
+    current_stock = line.variant.stocks.first()
+    line.quantity = current_stock.quantity - 1
+    line.save()
+
+    variables = {
+        "token": checkout.token,
+        "lines": [{"variantId": variant_id, "quantity": current_stock.quantity - 1}],
```
Comment (fowczarek) [inline saleor/graphql/checkout/tests/test_checkout_lines.py:506]: In this case, you call mutation with the same quantity that is set in the checkout line.

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**